contracts/SportMarkets/SportPositions/SportPositionalMarketManager.sol

```diff
@@ -146,6 +146,16 @@ contract SportPositionalMarketManager is Initializable, ProxyOwned, ProxyPausabl
         }
     }
 
+    function getDoubleChanceMarkets(address market) external view returns (address[] memory) {
```

Comment:
why is this method needed since the variable is already public? 
`  mapping(address => address[]) public doubleChanceMarkets;`